tests/src/test/java/alluxio/shell/command/ChownCommandTest.java

```diff
@@ -39,6 +63,65 @@ public void chown() throws IOException, AlluxioException {
     Assert.assertEquals("user2", owner);
   }
 
+  @Test
+  public void chownValidOwnerValidGroup() throws Exception {
+    setupTestGroupMappingService();
+    FileSystemTestUtils.createByteFile(mFileSystem, "/testFile", WriteType.MUST_CACHE, 10);
+    String newOwner = "alice";
+    List<String> groups = CommonUtils.getGroups(newOwner);
+    Random rand = new Random();
+    int numOfGroups = groups.size();
+    for (int i = 0; i < 8; i++) {
+      int k = rand.nextInt(numOfGroups);
+      String group = groups.get(k);
+      mFsShell.run("chown", newOwner + ":" + group, "/testFile");
+      checkFileOwnerAndGroup("/testFile", newOwner, group);
+    }
+  }
+
+  @Test
+  public void chownInvalidOwnerValidGroup() throws Exception {
+    setupTestGroupMappingService();
+    FileSystemTestUtils.createByteFile(mFileSystem, "/testFile", WriteType.MUST_CACHE, 10);
+    String newOwner = "alice";
+    String nonexistUser = "nonexistuser";
+    List<String> groups = CommonUtils.getGroups(newOwner);
+    int numOfGroups = groups.size();
+    Random rand = new Random();
+    String originalOwner = mFileSystem.getStatus(new AlluxioURI("/testFile")).getOwner();
+    String originalGroup = mFileSystem.getStatus(new AlluxioURI("/testFile")).getGroup();
+    for (int i = 0; i < 8; i++) {
```

Comment:
Because potentially in the future we may rewrite this test such that a user can belong to multiple groups. I want this to test multiple randomly-chosen groups.